Week2/homework/map-filter.js

```diff
@@ -1,8 +1,12 @@
 'use strict';
 
 function doubleOddNumbers(numbers) {
-  // Replace this comment and the next line with your code
+  const oddNumbers = numbers.filter(elem => elem % 2);
+  const doubledNumbers = oddNumbers.map((elem, index, arr) =>
```

Comment:
Ah - I don't think we need to write our `map` function like this. Since we have already filtered out odd numbers on line 4 using the `filter` method, when we run the map function on `oddNumbers` array, we are already sure that all the numbers inside it are odd. Thus, we can skip the check `elem % 2 !== 0 ? arr[index] * 2 : elem,` and simply write it as `oddNumbers.map(elem => elem * 2)` 🙂